tests/unit/s2n_server_new_session_ticket_test.c

```diff
@@ -303,4 +304,181 @@ int main(int argc, char **argv)
             EXPECT_EQUAL(output, test_cases[i].expected_output);
         }
     }
+
+    /* s2n_tls13_server_nst_send */
```

Comment:
Can you add a test for sending 0 tickets (not bc it's not tls1.3 or something, but just because 0 tickets are requested)